### PR TITLE
Fixed homebrew openssl@1.1

### PR DESCRIPTION
**Pull request by catap**
A few change to fixed `OPENSSL_LIBS` and `OPENSSL_FLAGS` in general and at homebrew ;)

### CODE REVIEW

File: src/configure.ac [hunk 1]
```diff
@@ -602,7 +602,7 @@ else
       [AC_MSG_CHECKING([supplied paths for OpenSSL])]
       [JTR_LIST_ADD(CFLAGS, [$OPENSSL_CFLAGS])]
       [if test "x$cross_compiling" = "xyes" -a "x$OPENSSL_LIBS" = "x"; then
-        OPENSSL_LIBS="-lssl -lcrypto"
+        OPENSSL_LIBS="${OPENSSL_LIBS} -lssl -lcrypto"
```
Comment (magnumripper) [inline src/configure.ac:605]: This will only happen when `$OPENSSL_LIBS` is empty (per line 604) so it's a no-op change

**Review by magnumripper — APPROVED** [review]

**Comment by magnumripper** [comment]
Thanks!

File: src/configure [hunk 2]
```diff
@@ -5222,11 +5222,15 @@ fi
...
-   ADD_LDFLAGS="$ADD_LDFLAGS -L/usr/local/opt/openssl/lib"
+if test -z "OPENSSL_LIBS"; then
+  if test -d /usr/local/opt/openssl/lib; then
+     ADD_LDFLAGS="$ADD_LDFLAGS -L/usr/local/opt/openssl/lib"
+  fi
 fi
-if test -d /usr/local/opt/openssl/include; then
-   ADD_CFLAGS="$ADD_CFLAGS -I/usr/local/opt/openssl/include"
+if test -z "OPENSSL_CFLAGS"; then
+  if test -d /usr/local/opt/openssl/include; then
```
Comment (magnumripper) [inline src/configure:5231]: Both checks above are missing the `$` in `$OPENSSL_(...)`. Fixed in 695d367b

Comment (catap) [reply src/configure:5231]: Yay

**Pull request merged**